nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -399,11 +349,17 @@ contract StakingEscrow is Upgradeable, IERC900History {
     //------------------ ERC900 connectors ----------------------
 
     function totalStakedForAt(address _owner, uint256 _blockNumber) public view override returns (uint256) {
-        return stakerInfo[_owner].history.getValueAt(_blockNumber);
+        if (isUpgrade == UPGRADE_TRUE) {
+            return stakerInfo[_owner].history.getValueAt(_blockNumber);
+        }
+        return 0;
```

Comment:
Great trick! Does this mean that in the next upgrade, we can just return 0 always?

Reply:
yep, see #2825